bedrock/security/templates/security/web-bug-bounty.html

```diff
@@ -100,7 +100,7 @@ <h2 id="payouts-section">Payouts</h2>
     <li>Includes IDORs that bypass authentication or authorization for significant actions</li>
     <li>Significant actions only, such as changing email/passwords, deleting accounts, etc.</li>
     <li>Must be able to conduct significant action (i.e., not defacement, phishing, cookie injection, etc.)</li>
-    <li>For domains falling outside *.mozilla.org, *.mozilla.com, *.mozilla.net, and *.firefox.com</li>
+    <li>For domains falling outside *.mozilla.org, *.mozilla.com, *.mozilla.net, and *.firefox.com. *.ngrok.io and similar development-only domains are excluded.</li>
```

Comment:
I think we may be overloading on the exclusion here...

I think intent is:

$500 for *.mozilla.org, *.mozilla.com, *.mozilla.net, and *.firefox.com takeovers
$200 for anything else excluding *.ngrok.io and similar development-only domains takeovers
 
What if we put in two exceptions, each with their own number and put the number next to 500 and 200 respectively?   Would that make our intent more clear?

Reply:
Yeah good point. Will make that change.